src/ensembl/src/shared/toggle-image-button/ToggleImageButton.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import dafaultStyles from './ToggleImageButton.scss';
+import ImageHolder from './ImageHolder';
+
+export enum ImageButtonStatus {
+  ACTIVE = 'active',
+  INACTIVE = 'inactive',
+  DISABLED = 'disabled'
+}
+
+type Props = {
+  buttonStatus: ImageButtonStatus;
+  description?: string;
+  image: any;
+  imageStyles?: any;
+  onClick: () => void;
+};
+
+const ToggleImageButton = (props: Props) => {
+  return (
+    <div
```

Comment:
Should it be a div or a button? @imran-ebi ?

Reply:
I think it should be a button

Reply:
That's done.